oviewer/action.go

```diff
@@ -244,8 +244,8 @@ func (root *Root) goLineNumber(lN int) {
 	root.setMessagef("Moved to line %d", lN+1)
 }
 
-// markNext moves to the next mark.
-func (root *Root) markNext(context.Context) {
+// nextMark moves to the next mark.
+func (root *Root) nextMark(context.Context) {
```

Comment:
I wouldn't rename them in the same logic as

https://github.com/noborus/ov/pull/623#discussion_r1749265333

But why not

Reply:
This depends on the naming rules of the function name.

Reply:
as I said, why not :grin: